.github/workflows/documentation.yml

```diff
@@ -0,0 +1,35 @@
+name: Cargo Create Docs
+
+on:
+  push:
+    branches: [nik-add-docs-in-CI]
```

Comment:
Just modify this to "main" and should be good for merge :+1:

Reply:
still testing.. this is not working